test/system/helpers.bash

```diff
@@ -137,11 +137,16 @@ function run_podman() {
         '?')             expected_rc=  ; shift;;  # ignore exit code
     esac
 
+    local BIN=$PODMAN
+    if [[ ! -z "$COVERAGE" ]]; then
```

Comment:
I find positive tests more readable than negative: `-n` is better here
```bash
    if [[ -n "$COVERAGE" ]]; then
```

Reply:
Much better, thanks!